Library/Homebrew/cmd/help.rb

```diff
@@ -39,4 +39,24 @@ def help
   def help_s
     HOMEBREW_HELP
   end
+
+  def help_cmd(cmd)
+    if File.exist?(HOMEBREW_LIBRARY_PATH/"cmd/#{cmd}.sh")
+      cmd_path = HOMEBREW_LIBRARY_PATH/"cmd/#{cmd}.sh"
+    elsif ARGV.homebrew_developer? && File.exist?(HOMEBREW_LIBRARY_PATH/"dev-cmd/#{cmd}.sh")
+      cmd_path = HOMEBREW_LIBRARY_PATH/"dev-cmd/#{cmd}.sh"
+    elsif File.exist?(HOMEBREW_LIBRARY_PATH/"cmd/#{cmd}.rb")
+      cmd_path = HOMEBREW_LIBRARY_PATH/"cmd/#{cmd}.rb"
+    elsif ARGV.homebrew_developer? && File.exist?(HOMEBREW_LIBRARY_PATH/"dev-cmd/#{cmd}.rb")
+      cmd_path = HOMEBREW_LIBRARY_PATH/"dev-cmd/#{cmd}.rb"
+    else
+      return ""
+    end
+
+    cmd_path.read.
+      split("\n").
+      grep(/^#:/).
+      map { |line| line.slice(2..-1).delete("`").sub(/^  \* /, "") }.
```

Comment:
I think it's nicer without the bullet points and the big number of backticks that we need to get the formatting of the man page right definitely feels off on the CLI.